### ISSUE TITLE
Question: Is it expected that every time I respawn the image after restarting the server, the content of the image is recreated, even in the same day?

### ISSUE THREAD

**Issue by pacospace** [AICoE/s2i-custom-notebook#41]
![Screenshot from 2021-07-01 16-39-20](https://user-images.githubusercontent.com/27498679/124142996-08ed2280-da8b-11eb-8e83-2541e22e7efc.png)

**Comment by 4n4nd** [comment]
yep this is designed to copy over the repo files on every spawn of the image. This is done [here](https://github.com/AICoE/s2i-custom-notebook/blob/master/s2i/bin/run#L4-L13).

**Comment by 4n4nd** [comment]
Maybe you can update the script to add a flag that can enable/disable this feature.

For example, it could check for an env variable `S2I_CLONE_REPO=true` before it copies the repo files over.
